e2e/app/src/App.tsx

```diff
@@ -9,10 +9,10 @@ import {
   BandwidthLimit,
   SimulcastConfig,
 } from '@fishjam-dev/ts-client';
-import { PeerMessage } from './protos/fishjam/peer_notifications';
 import { useEffect, useState, useSyncExternalStore } from 'react';
 import { MockComponent } from './MockComponent';
 import { VideoPlayerWithDetector } from './VideoPlayerWithDetector';
+import { PeerMessage } from '../../../src/protos/fishjam/peer_notifications';
```

Comment:
Is this correct import? I think that it should be imported via `@fishjam-dev/ts-client` path

Reply:
What do you think about hiding `protos` inside an additional namespace?
```
import { PeerMessage } from '@fishjam-dev/ts-client/protos';
```

Reply:
> What do you think about hiding protos inside an additional namespace?

Good idea :)